docs/adr/0002-project-locks.md

```diff
@@ -0,0 +1,160 @@
+
+# ADR-001: Project Locks
+
+- Date: 2023-04-18 
+- Status: Draft 
```

Comment:
Since this is in a draft state, it may make sense to put the 0001 adr in a separate pr since that will be faster to review, approve, and merge while this 0002 gets reviewed

Reply:
I'm actually of the opinion to merge the draft, I don't want to be the only gatekeeper/proposer of solutions and allow other maintainers/community members to contribute changes to the doc before it is accepted.

Reply:
I do not think we need a new PR, This PR and its comments are the histories for which this process of ADRs where created etc

Reply:
Merging the draft is antithetical to adrs which are immutable. It basically sets the draft as accepted by putting it into the codebase... It will also ensure that it will never be touched again.

Reply:
No one is saying we will do this on all the ADRs, this is the first one, new process, we are learning that is all

Reply:
Ok I'll resolve this. I defer to you folks